mobile/src/main/java/ch/epfl/sdp/mobile/ui/i18n/English.kt

```diff
@@ -10,4 +10,7 @@ object English : LocalizedStrings {
   override val authenticateSwitchToLogIn = "Log in with email"
   override val authenticateSwitchToRegister = "Register with email"
   override val authenticateErrorFailure = "Something went wrong when authenticating"
+
+  override val sectionSocial = "Players"
+  override val sectionSettings = "Settings"
```

Comment:
Do we actually specify what language to use in the app ? Or is it implicitly done ?

Reply:
We don't have more than English yet, but we have all the infrastructure required for languages that change based on device configuration (see `LocalLocalizedStrings`).

Reply:
👍